File: assign-3/prime-or-not.c
#include <stdio.h>

void primecheck(int no);
int main()
{
    primecheck(6);
}

void primecheck(int no)
{
    int count=0;
    for (int i = 1; i <=no; i++)
    {
        if (no%i==0)
        {
            count++;
        }
        
    }
    if (count>2)
    {
        printf("this is not prime no");
    }
    else
    {
        printf("this is prime no ");
    }
    
    
}

File: Java/primeOrNot.java
package practise;

import java.util.Scanner;

public class primeOrNot {

	public static void main(String[] args) {
		int count=0;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter a number:");
		int number = sc.nextInt();

		for (int i = 2; i <number ;i++) {
			if (number%i==0) {
			count=count+1;	
			}
		}
		if (count==0) {
			System.out.println("no is prime");
		}
		else {
			System.out.println("no is not prime");
		}
	}

}
